components/tree/private/branch.jsx

```diff
@@ -182,12 +182,12 @@ const handleKeyDownEnter = (event, props) => {
 const handleKeyDown = (event, props) => {
 	mapKeyEventCallbacks(event, {
 		callbacks: {
```

Comment:
@garygong I'm having some strange behavior around tabindexes. If the selection is the first item, things act as expected. If you select any other item however, the first item maintains its tabindex, adding an extra tab stop to the tree.

Reply:
Thanks for finding this.  It turns out that this was caused by a typo in the code!  It's fixed now and the new link is here: https://design-system-react-co-pr-1323.herokuapp.com/?selectedKind=SLDSTree&selectedStory=Base&full=0&addons=1&stories=1&panelRight=0&addonPanel=storybook%2Factions%2Factions-panel

Let me know if you see anything else.  Thanks!

Reply:
I don't think I understand the Highlighted Search variant. Should the tree get filtered down to only nodes which match the search? When I type in the search box, my focus is moved to the tree as soon as I press a key. I was able to type "nut", but I had to shift tab back to the input twice to add letters.